src/components/Dialog/Dialog.visual.tsx

```diff
@@ -32,15 +33,43 @@ const DialogWithContent = (props) => (
   </Dialog>
 );
 
-visualize('Dialog', () => {
-  story('simple', () => {
-    snap('default props', <Dialog isOpen />);
-    snap('Dialog with some content', <DialogWithContent />);
+const runDesktopSnapshots = (isRtl) => {
+  story('Basic', () => {
+    snap(
+      `Desktop with default props  ${isRtl ? 'rtl' : ''}`,
+      TPAComponentsWrapper({ mobile: false, rtl: isRtl })(<Dialog isOpen />),
+    );
+    snap(
+      `Desktop Dialog with some content  ${isRtl ? 'rtl' : ''}`,
+      TPAComponentsWrapper({ mobile: false, rtl: isRtl })(
+        <DialogWithContent />,
+      ),
+    );
+  });
+};
+
+const runMobileSnapshots = (isRtl) => {
+  story('Basic', () => {
+    snap(
+      `Mobile with default props  ${isRtl ? 'rtl' : ''}`,
+      TPAComponentsWrapper({ mobile: true, rtl: isRtl })(<DialogWithContent />),
+    );
+    snap(
+      `Mobile with not fullscreen version  ${isRtl ? 'rtl' : ''}`,
+      TPAComponentsWrapper({ mobile: true, rtl: isRtl })(
+        <DialogWithContent notFullscreenOnMobile />,
+      ),
+    );
   });
```

Comment:
Nice!
In order to see how the visual tests story page you could run:
`npm run eyes-storybook-debug`

When you run it you would be able to see that the mobile tests that you added are not really rendered as they would be on mobile.

We could use the `MobileExample.tsx` file in the visual tests in order to use the mobile iframe. (this is the mobile iframe that we use on the Storybook).
https://github.com/wix/wix-ui-tpa/blob/master/stories/utils/MobileExample.tsx

In order to import this file from the `Dialog.visual.tsx` file we should add require in the `.storybook-test/config.js`.
https://github.com/wix/wix-ui-tpa/blob/819117845ec1a142103818720a1dd7f4cc0e473a/.storybook-test/config.js#L7
` require('stories/utils/MobileExample');`

Reply:
ah nice, I saw that it's not really rendered as mobile, checked some components and didn't find anything I could use to simulate mobile properly. Glad you showed this, I'll be able to add another test case to simulate edge case with small mobile screen height.